### ISSUE TITLE
Missing label in login page for accessibility.

### ISSUE THREAD

**Issue by r-shaw** [edwinsoto88/color-vision#10]
After running the WAVE accessibility tool on the login page I found there were 2 missing labels for accessibility purposes, so I need to add the labels.
